csodiaq/identifier/Identifier.py

```diff
@@ -0,0 +1,186 @@
+from csodiaq.loaders.library.LibraryLoaderContext import LibraryLoaderContext
+from csodiaq.loaders.query.QueryLoaderContext import QueryLoaderContext
+from csodiaq.identifier.poolingFunctions import generate_pooled_library_and_query_spectra_by_mz_windows
+from csodiaq.identifier.matchingFunctions import match_library_to_query_pooled_spectra, eliminate_low_count_matches, eliminate_matches_below_fdr_cutoff
+from csodiaq.identifier.scoringFunctions import score_library_to_query_matches, identify_all_decoys, determine_index_of_fdr_cutoff, filter_matches_by_ppm_offset_and_tolerance, calculate_ppm_offset_tolerance
+from csodiaq.identifier.outputWritingFunctions import extract_metadata_from_match_and_score_dataframes, format_output_line, format_output_as_pandas_dataframe
+import pandas as pd
+
+class Identifier():
+    """
+    Class for identifying library peptides in input query spectra for mass spec DIA experiments.
+
+    Extended Summary
+    ----------------
+    The purpose of this class is to consolidate the primary workflow of identifying peptides
+        from a library file in mass spec DIA query files. This includes finding similar peaks
+        between them both (matching) and eliminating false positive matches (scoring). There is
+        an optional correction step that further eliminates false positive library-query peak
+        matches, refining both the final scores and false positive peptide elimination.
+
+    Attributes
+    ----------
+    _args : dictionary
+        This dictionary references the arguments inserted by the end user.
+    _libraryDict : dictionary
+        A standardized representation of the library input file for use of the program.
+            It is the output of the LibraryLoaderContext class.
+    _decoySet : set
+        A set containing all the keys of self._libraryDict that represent a decoy insert.
+            Decoys are used to calculate the probability that a non-decoy match is a
+            false positive.
+    """
+
+    def __init__(self, args):
+        self._args = args
+        self._libraryDict = LibraryLoaderContext(self._args["library"]).load_csodiaq_library_dict()
+        self._decoySet = set([key for key,value in self._libraryDict.items() if value["isDecoy"]])
```

Comment:
the 'args' parameter is the input from argsparse, which contains commands/flags issued by the user from the command line. Are you just recommending I change the name to "commandLineArgs" or some such to avoid the standard use of 'args' as a function parameter in python?

Reply:
We talked, I'll just rename the parameter